helm-charts/basehub/values.yaml

```diff
@@ -590,15 +590,37 @@ jupyterhub:
                 matchLabels:
                   app.kubernetes.io/component: traefik
   hub:
-    loadRoles:
-      # Should use this, not hub.config.JupyterHub.load_roles - that will
-      # override any existing load_roles set by z2jh
-      service-use:
-        name: user
-        scopes:
-          # Allow all users access to 'services', which includes dask-gateway & configurator
-          - access:services
-          - self
+    # hub.loadRoles is z2jh native config to enable configuration of
```

Comment:
yay that it has no effect and we can remove it.

I'd suggest we just simply fully remove it, and not actually have this large comment here. It adds additional things that must be kept up to date whenever things elsewhere change, and makes this already large file even larger. Let's just remove that?